tyssue/draw/plt_draw.py

```diff
@@ -119,57 +119,111 @@ def create_gif(
         shutil.rmtree(graph_dir)
 
 
-def sheet_view(sheet, coords=COORDS, ax=None, **draw_specs_kw):
-    """Base view function, parametrizable
+def sheet_view(sheet, coords=COORDS, ax1=None, ax2=None, **draw_specs_kw):
```

Comment:
it would be better to keep the first `ax` argument named `ax` and keep ax2 with None by default or name `ax2`  `cmap_ax` or something equivalent. This would avoid breaking the API. In many scripts I pass the `ax` argument explicitly, they would be broken by the merge.